packages/terra-worklist-data-grid/tests/jest/WorklistDataGrid.test.jsx

```diff
@@ -132,6 +132,108 @@ describe('basic grid', () => {
 
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('verifies row selection when space is pressed on a masked cell', () => {
```

Comment:
should these tests be in this PR? They look like duplicates of https://github.com/cerner/terra-framework/pull/1743/files#diff-65175b0cfb2569e3acf7c9419b1e0bc9decc7e2b22c297a9519b867784bdb586R136

Reply:
Now that I have merged that pull request to main, I will rebase.  I was trying to get the code out for review simultaneously.  I apologize for the confusion on this one.